### ISSUE TITLE
AX-026-Aria-Labels-Missing-Grid

### ISSUE THREAD

**Issue by aestrada7** [aestrada7/accounting-js#30]
The editable grid is missing aria-labels

**Comment by aestrada7** [comment]
Completed on 653117a6
